README.md

```diff
@@ -97,7 +98,7 @@ vcs import src < ros2_dotnet_uwp.repos
 cd \dev\ament
 call install\local_setup.bat
 cd \dev\ros2
-colcon build --merge-install --cmake-args -A "%TARGET_ARCH%" -DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION=10.0.14393 -DTHIRDPARTY=ON -DINSTALL_EXAMPLES=OFF -DBUILD_TESTING=OFF
+colcon build --merge-install --packages-ignore rcl_logging_log4cxx --cmake-args -A "%TARGET_ARCH%" -DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION=10.0.17763 -DTHIRDPARTY=ON -DINSTALL_EXAMPLES=OFF -DBUILD_TESTING=OFF
```

Comment:
Would this version work with the Hololens? I used 14393 because at the time, the Hololens used a rather old version of the Windows SDK.

Reply:
Awesome, thanks! Then that means that the C# side can also use newer features?

Reply:
I recall having to use some old C# API, but unfortunately I can't remember where the code was. I think it had to do with reflection or something like that.